packages/table/src/table.tsx

```diff
@@ -402,6 +403,52 @@ export class Table extends AbstractComponent2<TableProps, TableState, TableSnaps
         this.quadrantStackInstance.scrollToPosition(correctedScrollLeft, correctedScrollTop);
     }
 
+    /**
```

Comment:
I'd prefer to not add code to Table in this PR, just limit the changes to Table2

Reply:
ack that checks out